source/common/tls/server_context_impl.h

```diff
@@ -82,12 +92,12 @@ class ServerContextImpl : public ContextImpl, public Envoy::Ssl::ServerContext {
                          unsigned int inlen);
   int sessionTicketProcess(SSL* ssl, uint8_t* key_name, uint8_t* iv, EVP_CIPHER_CTX* ctx,
                            HMAC_CTX* hmac_ctx, int encrypt);
-  bool isClientEcdsaCapable(const SSL_CLIENT_HELLO* ssl_client_hello);
-  bool isClientOcspCapable(const SSL_CLIENT_HELLO* ssl_client_hello);
   OcspStapleAction ocspStapleAction(const Ssl::TlsContext& ctx, bool client_ocsp_capable);
 
   SessionContextID generateHashForSessionContextId(const std::vector<std::string>& server_names);
 
+  Ssl::TlsCertificateSelectorFactoryCb tls_certificate_selector_factory_cb_;
```

Comment:
I don't think you need to store this anymore; it should just be needed in the constructor now